src/Button/index.js

```diff
@@ -32,7 +32,7 @@ Button.defaultProps = {
   busy: null,
   className: null,
   disabled: false,
-  iconLink : false
+  iconLink: false
```

Comment:
The attribute should be just `icon`, instead `iconLink`.  Why only links can have icons? Let's make this functionality more generical.